examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -4,82 +4,44 @@
 package e2e_test
 
 import (
-	"context"
 	"encoding/json"
-	"fmt"
-	"math"
 	"testing"
 
-	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/tests/fixture/e2e"
 	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
 	"github.com/stretchr/testify/require"
 
-	"github.com/ava-labs/hypersdk/auth"
-	"github.com/ava-labs/hypersdk/chain"
-	"github.com/ava-labs/hypersdk/codec"
-	"github.com/ava-labs/hypersdk/crypto/ed25519"
-	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/consts"
-	"github.com/ava-labs/hypersdk/examples/morpheusvm/genesis"
-	"github.com/ava-labs/hypersdk/fees"
-	"github.com/ava-labs/hypersdk/rpc"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/tests/workload"
 	"github.com/ava-labs/hypersdk/tests/fixture"
-	"github.com/ava-labs/hypersdk/tests/workload"
 
-	lrpc "github.com/ava-labs/hypersdk/examples/morpheusvm/rpc"
 	he2e "github.com/ava-labs/hypersdk/tests/e2e"
 	ginkgo "github.com/onsi/ginkgo/v2"
 )
 
 const owner = "morpheusvm-e2e-tests"
 
-var (
-	_            workload.TxWorkloadFactory  = (*workloadFactory)(nil)
-	_            workload.TxWorkloadIterator = (*simpleTxWorkload)(nil)
-	flagVars     *e2e.FlagVars
-	genesisBytes []byte
-	factory      *auth.ED25519Factory
-)
+var flagVars *e2e.FlagVars
 
 func TestE2e(t *testing.T) {
 	ginkgo.RunSpecs(t, "morpheusvm e2e test suites")
 }
 
 func init() {
 	flagVars = e2e.RegisterFlags()
+}
+
+var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
```

Comment:
What's this doing?

Reply:
Added comments here https://github.com/ava-labs/hypersdk/pull/1303/commits/6119be853631bc0a2e7a3991df8adf089ca20a35